tests/conftest.py

```diff
@@ -39,6 +144,11 @@ def app_config(app_config):
     ] = "invenio_jsonschemas.proxies.current_refresolver_store"
     app_config["CACHE_TYPE"] = "SimpleCache"
 
+    app_config["ENTITY_REFERENCE_UI_RESOLVERS"] = {
+        "user": user_entity_reference_ui_resolver,
+        "thesis_draft": draft_record_entity_reference_ui_resolver,
+    }
+
```

Comment:
usage example, why "thesis_draft"? Do we need to generate this to invenio.cfg?

Reply:
it's mapped to type_id similarly how invenio resolvers use type_key to map to type_id

it could be generated in model builder instead i guess